lib/index/index.tsx

```diff
@@ -1,27 +1,21 @@
-import { Container, Typography } from '@mui/material';
 import React from 'react';
 import { CelebGallery } from '~/lib/components/CelebGallery';
 import { sanityClient } from '~/lib/components/sanityio';
-import s from './styles.module.scss';
 import { top100CelebSlugs } from './top100CelebSlugs';
 
 export const Index = (p: any) => {
   return (
-    <>
-      <div className={s.fancy}>
-        <div className={s.fancyBackground} />
-      </div>
-
-      <Container className={s.Index} maxWidth="md">
-        <Typography variant="h1" fontWeight={400}>
-          Top 100 celebs
-        </Typography>
+    <main className="mx-auto max-w-home-container">
+      <div className="container mx-auto px-4 md:px-0">
```

Comment:
why do we need to remove the x-padding in screens larger than `md`?

and do we need a `div` directly inside `main`? Can this mark-up be simplified?

Reply:
I have the same answer here in this comment about to x-padding, https://github.com/hollowverse/hollowverse/pull/69#discussion_r859377201